### PR TITLE
* #94: Updated to the `virtual-schema-common-java:15.0.0.`

### LINKED ISSUE #94: Update to virtual-schema-common-java :15.0.0

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #94

### CODE REVIEW

File: doc/changes/changes_9.0.1.md [hunk 1]
```diff
@@ -5,10 +5,12 @@ Code name:
 ## Bug Fixes
 
 * #92: Fixed base classes of QueryRewriter.
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_9.0.1.md:7]:
```suggestion
* #92: Fixed base classes of `QueryRewriter`.
```

File: src/test/java/com/exasol/adapter/dialects/SqlGenerationVisitorTest.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ void testVisitSqlFunctionScalarCast() throws AdapterException {
         final DataType dataType = DataType.createDecimal(18, 0);
         final SqlLiteralDouble argument = new SqlLiteralDouble(20);
         final SqlFunctionScalarCast sqlFunctionScalarCast = new SqlFunctionScalarCast(dataType, argument);
-        assertThat(sqlGenerationVisitor.visit(sqlFunctionScalarCast), equalTo("CAST(20.0 AS DECIMAL(18, 0))"));
+        assertThat(sqlGenerationVisitor.visit(sqlFunctionScalarCast), equalTo("CAST(2E1 AS DECIMAL(18, 0))"));
```
Comment (chiaradiamarcelo) [inline src/test/java/com/exasol/adapter/dialects/SqlGenerationVisitorTest.java:47]: Is this required cause of https://github.com/exasol/virtual-schema-common-java/pull/210 ?

File: src/test/java/com/exasol/adapter/jdbc/JDBCAdapterIT.java [hunk 3]
```diff
@@ -27,14 +26,14 @@ void testRegisteredAdapterFactories() throws AdapterException {
...
-                + "            \"" + SQL_DIALECT_PROPERTY + "\" : \"DERBY\"\n," //
                 + "            \"" + CONNECTION_NAME_PROPERTY + "\" : \"DERBY_CONNECTION\"\n" //
                 + "        }\n" //
                 + "    }\n" //
                 + "}";
         final ExaMetadata exaMetadata = Mockito.mock(ExaMetadata.class);
         RequestDispatcher.adapterCall(exaMetadata, rawRequest);
-        final List<AdapterFactory> registeredFactories = AdapterRegistry.getInstance().getRegisteredAdapterFactories();
-        assertThat(registeredFactories, hasItem(instanceOf(JDBCAdapterFactory.class)));
+        final ServiceLoader<AdapterFactory> adapterFactoryLoader = ServiceLoader.load(AdapterFactory.class);
```
Comment (chiaradiamarcelo) [inline src/test/java/com/exasol/adapter/jdbc/JDBCAdapterIT.java:35]: this is OK for now, eventually we have to implement a more thorough integration test. Please create a ticket for it if there.

**Review by chiaradiamarcelo — DISMISSED** [review]

File: src/test/java/com/exasol/adapter/dialects/SqlGenerationVisitorTest.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/dialects/SqlGenerationVisitorTest.java:47]: Yes, we changed the format of double and exactnumeric

File: src/test/java/com/exasol/adapter/jdbc/JDBCAdapterIT.java (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/jdbc/JDBCAdapterIT.java:35]: Done https://github.com/exasol/virtual-schema-common-jdbc/issues/96

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**